st2common/st2common/models/db/stormbase.py

```diff
@@ -13,10 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import bson
 import datetime
```

Comment:
It doesn't, I just sorted it so it follows order from other modules (stdlib, 3rdparty, local).